app/jobs/modsulator_job.rb

```diff
@@ -121,6 +115,40 @@ def update_metadata(druid, xml_string, log)
     end
   end
 
+  # Open a new version for the given object if it is in the accessioned state.
+  # @param   [Dor::Item]  dor_object          The object to version
+  # @param   [String]     original_filename   The name of the uploaded file
+  # @param   [String]     user_login          The current user_login
+  # @param   [File]       log                 Log file handle
+  def version_object(dor_object, original_filename, user_login, log)
+    if(accessioned(dor_object))
+      if !DorObjectWorkflowStatus.new(dor_object.pid).can_open_version?
```

Comment:
don't care about unless vs if, personally, but line 124 is missing the space after the if ...